Library/Aliases/neo4j-client.rb

```diff
@@ -0,0 +1 @@
+../Formula/libneo4j-client.rb
```

Comment:
Any reason to have this alias? `neo4j-client` already matches `libneo4j-client`.